README.md

```diff
@@ -72,12 +72,9 @@ ng generate component my-new-component
 ng g component my-new-component # using the alias
 
 # components support relative path generation
-# if in the directory src/app/feature/ and you run
```

Comment:
Instead of deleting these, I suggest you just add your example as the first example.